### PR TITLE
Update pointer to RS including refactoring input field schema.

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**
FYI: @dephillipsmichael

### CODE REVIEW

File: BridgeApp/BridgeApp/Data Tracking/Model/SBARemoveTrackedItemStepObject.swift [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    
+    /// Initialize from a `Decoder`.
+    ///
+    /// - example:
+    /// ```
+    ///    let json = """
+    ///        {
+    ///            "identifier": "foo",
+    ///            "type": "selection",
+    ///            "title": "Are you sure you want to remove the items",
```
Comment (Erin-Mounts) [inline BridgeApp/BridgeApp/Data Tracking/Model/SBARemoveTrackedItemStepObject.swift]: should there be a question mark after 'items'?

File: BridgeApp/BridgeApp/Data Tracking/Model/SBATrackedItemRemindersStepObject.swift [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        guard let subclassCopy = copy as? SBATrackedItemRemindersStepObject else {
+            assertionFailure("Superclass implementation of the `copy(with:)` protocol should return an instance of this class.")
+            return
+        }
+        subclassCopy.modalTitle = self.modalTitle
+        subclassCopy.descriptionFormat = self.descriptionFormat
+        subclassCopy.noReminderSetText = self.noReminderSetText
+    }
+    
+    /// Returns the reminder choice step
```
Comment (Erin-Mounts) [inline BridgeApp/BridgeApp/Data Tracking/Model/SBATrackedItemRemindersStepObject.swift]: .

File: BridgeApp/BridgeApp/Data Tracking/Model/SBATrackedItemRemindersStepObject.swift [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    }
+    
+    func updateAnswer(from modalTaskPath: RSDTaskPath, with stepIdentifier: String?) {
+        if let modalStepIdentifier = stepIdentifier,
+            let collectionResult = modalTaskPath.result.findResult(with: modalStepIdentifier) as? RSDCollectionResultObject {
+            return self.updateAnswer(to: collectionResult.copy(with: self.step.identifier))
+        }
+    }
+    
+    /// - parameter result: The most recent result
```
Comment (Erin-Mounts) [inline BridgeApp/BridgeApp/Data Tracking/Model/SBATrackedItemRemindersStepObject.swift]: there don't actually appear to be any parameters...

File: BridgeApp/BridgeApp/Data Tracking/iOS/SBARemoveTrackedItemStepViewController.swift [hunk 4]
```diff
@@ -62,6 +62,7 @@ open class SBARemoveTrackedItemStepViewController: RSDStepViewController {
         if let removeStep = self.removeTrackedItemStep,
             let title = removeStep.title {
             if let underlinedSegment = removeStep.underlinedTitleSegment {
+                // TODO: syoung 06/23/2018 Refactor to use HTML for underlining and to use a placeholder such as "%1$@" for range replacement.
```
Comment (dephillipsmichael) [inline BridgeApp/BridgeApp/Data Tracking/iOS/SBARemoveTrackedItemStepViewController.swift]: This sounds promising.  Is there an example of this in the SDK?

File: BridgeApp/BridgeApp/Data Tracking/Model/SBARemoveTrackedItemStepObject.swift (hunk 1 above)
Comment (dephillipsmichael) [reply BridgeApp/BridgeApp/Data Tracking/Model/SBARemoveTrackedItemStepObject.swift]: This is a bad example on my part. I'll fix it in the branch I'm sending out soon.

File: BridgeApp/BridgeApp/Data Tracking/iOS/SBARemoveTrackedItemStepViewController.swift (hunk 4 above)
Comment (syoung-smallwisdom) [reply BridgeApp/BridgeApp/Data Tracking/iOS/SBARemoveTrackedItemStepViewController.swift]: No, I think that there might be an issue to include support for HTML "attributed text" tags but its not a feature that is currently supported by the Research framework.

**Comment by syoung-smallwisdom** [comment]
@Erin-Mounts 
This is code that @dephillipsmichael  is currently working on cleaning up. I just moved some of it and fixed the references to deprecated ui hints and "dataType" fields.

**Pull request merged**